readingDataPointsFilters:
- MaxDensityDataPointsFilter:
    maxDensity: 20.0
#  - RandomSamplingDataPointsFilter:
#      prob: 0.1

referenceDataPointsFilters:
#  - MaxDistDataPointsFilter:
#      maxDist: 20.0
  - RandomSamplingDataPointsFilter:
      prob: 0.01

matcher:
  KDTreeMatcher:
    knn: 3
    maxDist: 10.0
    epsilon: 0

outlierFilters:
  - TrimmedDistOutlierFilter:
     ratio: 0.75
  - SurfaceNormalOutlierFilter:
     maxAngle: 1.57
  - GenericDescriptorOutlierFilter:
     source: reference
     descName: probabilityStatic

errorMinimizer:
  PointToPlaneErrorMinimizer

transformationCheckers:
  - DifferentialTransformationChecker:
      minDiffRotErr: 0.001
      minDiffTransErr: 0.01
      smoothLength: 2
  - CounterTransformationChecker:
      maxIterationCount: 40
  - BoundTransformationChecker:
      maxRotationNorm: 0.80
      maxTranslationNorm: 30.00

inspector:
#  VTKFileInspector
  NullInspector

logger:
#  FileLogger
  NullLogger
